apispec/core.py

```diff
@@ -387,3 +393,26 @@ def validate_openapi_version(openapi_version_str):
         )
 
     return openapi_version
+
+
+class YAMLDumper(yaml.Dumper):
+    pass
+
+    @staticmethod
```

Comment:
Does this need to be a staticmethod? If so, it should be consistent with the unicode representer method.

Reply:
Done.